README.md

```diff
@@ -42,7 +42,7 @@ Alternatively, use the below command, replacing `$VERSION`, `$OSTYPE`
 as required:
 
 ```shell-session
-$ curl -fSsL "https://github.com/cdr/coder-doctor/releases/latest/download/coder-doctor_${VERSION}_${OSTYPE}_${ARCH}.tar.gz" | tar -xzvf -
+$ curl -fSsL "https://github.com/coder/coder-doctor/releases/download/${VERSION}/coder-doctor_${VERSION}_${OSTYPE}_${ARCH}.tar.gz" | tar -xzvf -
```

Comment:
Thanks for this @stevek-pro !

It looks like we need to make one _very minor_ modification here:



```suggestion
$ curl -fSsL "https://github.com/coder/coder-doctor/releases/download/v${VERSION}/coder-doctor_${VERSION}_${OSTYPE}_${ARCH}.tar.gz" | tar -xzvf -
```

based on the way this URL is:

https://github.com/coder/coder-doctor/releases/download/v0.0.1/coder-doctor_0.0.1_linux_amd64.tar.gz

from the `v0.0.1` pre-release:

https://github.com/coder/coder-doctor/releases/tag/v0.0.1